app/controllers/admin/tutorials_controller.rb

```diff
@@ -4,15 +4,16 @@ def edit
   end
 
   def create
-    if results.snippet.nil?
-      flash[:error] = 'Sorry, that ID is not valid. Try again?'
-      redirect_to new_admin_youtube_playlist_path
+    if !missing_params.empty?
```

Comment:
I think instead of double negative you can just do `.present?` unless that was something rubofuck rejected?